ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -743,7 +743,16 @@ literals:
 - lost_power_alarm:
   - ACTIVE
   - INACTIVE
-
+- refrigerant_failed_alarm:
```

Comment:
shoud this be a `refrigerant_leak_alarm` ?

Reply:
Actually this is a refrigerant sensor fault alarm

Reply:
> shoud this be a `refrigerant_leak_alarm` ?
Actually this is a refrigerant sensor fault alarm

Reply:
Removing the refrigerant sensor failed alarm as cannot model it because sensor is a point type and cannot be used  as descriptor.